scripts/error-extractor/parseJSFile.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+var babylon = require('babylon');
+var fs = require('fs');
+
+function parseJSFile(filePath/* : string */)/* : Object */ {
+  var source = fs.readFileSync(filePath, 'utf8');
+  return babylon.parse(source, {
+    sourceType: 'module',
+    plugins: [
```

Comment:
Hey, thanks for catching this! This is kinda tricky -- as a parser, babylon has its own options [here](https://github.com/babel/babylon#plugins) and it's different from babel's. All our syntax (parser)-related configs are specified [here](https://github.com/facebook/fbjs/blob/master/babel-preset/configure.js#L34-L38) in `fbjs`. I'll update this one to make sure they are the same.